tests/fixtures/function.json

```diff
@@ -0,0 +1,13 @@
+{
+  "data": {
+    "resource": {
+      "object": {
+        "vars": {
+          "cluster_1": "join(\\n,data.template_file.cluster_1.*.rendered)",
+          "cluster_2": "join(\\n,data.template_file.cluster_2.*.rendered)",
```

Comment:
These are weird. Where did the quotes around `\n` and `name_%02d` go?

Reply:
I just noticed this comment was probably directed at me.  :)
It's not clear to me how those should look in json.  Should there be quotes there?  I assume they would have to be escaped if there were there, right?

Reply:
Yes, the quotes should be there, and they should be escaped. 

```
"cluster_1": "join(\"\\n\", data.template_file.cluster_1.*.rendered)"
```